shims/pom.xml

```diff
@@ -31,9 +31,50 @@
     <description>The RAPIDS SQL plugin for Apache Spark Shims</description>
     <version>21.10.0-SNAPSHOT</version>
 
-    <modules>
-        <module>${shim.module.name}</module>
-    </modules>
+    <profiles>
+        <!-- maven CLI and CI builds one at a time -->
+        <profile>
+            <id>with-classifier</id>
+            <activation>
+                <activeByDefault>true</activeByDefault>
+            </activation>
+            <modules><module>${shim.module.name}</module></modules>
+        </profile>
+
+        <!-- For easy switching in IDE -->
```

Comment:
I guess explicitly setting it makes ide happy?  can you see them all at once now?  perhaps a bit more explanation here

Reply:
I'll update, the main issue for IDEA is switching between differently named profiles that are activated by a single property value. I actually could not make it work even by adding as -D to the importer settings. So having consistently named profiles throughout submodules looks like a must here.